db/queries.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+const redisClient = require('redis').createClient;
+const postDb = require('./db.js');
+
+const redis = redisClient(6379, 'localhost');
+
+const db = mongoose.createConnection('mongodb://localhost/restaurant');
```

Comment:
you may want to pull this and the redis address into an environment variable and use `process.env.MONGO_URL` (or similar)